src/back/VectorBasedTile.js

```diff
@@ -18,7 +18,6 @@ VectorBasedTile.prototype._render = function (project, map, cb) {
         parse = function (data, resp) {
             try {
                 vtile.setData(data);
```

Comment:
In node-mapnik >= 3.5 `setData` does more work that previously. So I would recommend moving to the async version of `setData` for best performance. Before it did not matter. /cc @yohanboniface